pagopa/io-app
# fix: [IOBP-43] Removed now deprecated privative cards code

Depends on https://github.com/pagopa/io-app/pull/4611

## Short description
removal of privative cards from the app, although some tests still use them as a mock

## List of changes proposed in this pull request
- removal of privative cards' i18n keys
- removal of said feature's folder
- refactor of shared code whenever necessary

## How to test
`yarn jest`, also navigate to the wallet section and execute a full cobadge flow (io-dev-server is your friend here) to make sure nothing breaks.
the app should not be impacted aside from the fact that privative or "loyalty" cards have ceased to exist.
